modules/widgets/wordpress-post-widget.php

```diff
@@ -7,19 +7,65 @@
  * Author URI: http://automattic.com
  * License: GPL2
  */
+
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if (!defined('ABSPATH')) {
+	exit;
+}
+
 add_action( 'widgets_init', 'jetpack_display_posts_widget' );
 function jetpack_display_posts_widget() {
-	 register_widget( 'Jetpack_Display_Posts_Widget' );
+	register_widget( 'Jetpack_Display_Posts_Widget' );
 }
 
+
+/**
+ * Cron tasks
+ */
+
+/**
+ * Add a 10 minute interval to cron intervals.
+ */
+add_filter( 'cron_schedules', 'jetpack_display_posts_widget_cron_intervals' );
+function jetpack_display_posts_widget_cron_intervals() {
```

Comment:
This looks wrong.  A filter should accept a value, modify (or replace it), and then return it.

What would happen if another plugin were adding a `minutes_30` interval via the `cron_schedules` hook?